Podfile

```diff
@@ -21,6 +21,8 @@ end
 target 'podcasts' do
   platform :ios, app_ios_deployment_target.version
   common_pods
+  pod 'PulseCore', :git => 'https://github.com/kean/Pulse.git', :tag => '4.2.4', :configurations => %w(Debug Staging Prototype)
```

Comment:
<table data-meta="generated_by_pr-check">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Automattic/pocket-casts-ios/blob/456e5b941c35f04ba523a63c2f5644e9734320a8/Podfile#L24"></span>Style/HashSyntax: Use the new Ruby 1.9 hash syntax.</td>
    </tr>
  </tbody>
</table>